admin_guide/install/advanced_install.adoc

```diff
@@ -137,42 +160,202 @@ master.example.com openshift_node_labels="{'region': 'infra', 'zone': 'default'}
 node1.example.com openshift_node_labels="{'region': 'primary', 'zone': 'east'}"
 node2.example.com openshift_node_labels="{'region': 'primary', 'zone': 'west'}"
 ----
-endif::[]
-ifdef::openshift-origin[]
+====
+
+Modify the file to match your environment and specifications, and save it as
+*_/etc/ansible/hosts_*.
+
+[[single-master-multi-etcd-multi-node]]
```

Comment:
Why would we ever recommend this?  Sure you can do it, but I don't think we should publicly state it.  
You're either HA, or you're not.